src/Test/WordPress/WordPressAPITest.php

```diff
@@ -46,14 +58,46 @@ public function invokeMethod(&$object, $methodName, array $parameters = array())
 
     public function testGetDomainListReturnValue()
     {
-        $domainName = 'domainName.com';
-        $_SERVER['SERVER_NAME'] = $domainName;
+        $domainName = 'domain.com';
+
+        $this->mockDataStore = $this->getMockBuilder('CF\WordPress\DataStore')
+                ->disableOriginalConstructor()
+                ->setMethods(array('getDomainNameCache'))
+                ->getMock();
+
+        $this->mockDataStore->method('getDomainNameCache')->willReturn($domainName);
+
+        $this->wordpressAPI = new WordPressAPI($this->mockDataStore);
```

Comment:
Should move this into `setup()`.  Any changes you make to the `$mockDataStore` will still apply even after you've instantiated `$wordpressAPI`.